backend/app/assets/stylesheets/spree/backend/shared/_layout.scss

```diff
@@ -43,14 +43,45 @@
   }
 }
 
+// Page Layout
+//---------------------------------------------------
+
+html {
+  height: 100%;
+}
+
+body {
+  position: relative;
+  min-height: 100%;
+}
+
+.admin-nav {
+  @include position(fixed, 0 null 0 0);
```

Comment:
Wordpress uses Javascript to toggle between `position: absolute` and `fixed`, depdeny on the scroll position. Interesting approach. Like this. 

Do you mind to have some little more Javascript in the admin?